theforeman.org/yaml/includes/pulpcore_versions_copr.yaml.inc

```diff
@@ -0,0 +1 @@
+- 'nightly'
```

Comment:
We can include '3.39' here as well, since we are going to branch from `rpm/develop`

Reply:
I think that's out of scope for this PR: we should treat 3.39 as a separate branching step. We already know https://github.com/theforeman/theforeman-rel-eng/blob/master/procedures/pulpcore/update.md.erb is out of date, so please keep notes when you do.